util/test/tests/Vulkan/VK_Descriptor_Indexing.py

```diff
@@ -64,6 +64,7 @@ def check_capture(self):
             0: { 'dynamicallyUsedCount': 1, 'used': [15] },
             1: { 'dynamicallyUsedCount': 6, 'used': [4, 19, 20, 21, 49, 59] },
             2: { 'dynamicallyUsedCount': 2, 'used': [381, 386] },
+            3: { 'dynamicallyUsedCount': 0, 'used': [] },
```

Comment:
I think for this test to be robust we should ensure that one of the samplers in this descriptor set less than the count is actually used. Otherwise in future this test may not cover what we want it to e.g. if we statically determine that the shader doesn't use the descriptor set at all so we skip processing of it.